eng/common/scripts/Prepare-Release.ps1

```diff
@@ -184,6 +184,13 @@ else
   exit 1
 }
 
+$changelogIsValid = Confirm-ChangeLogEntry -ChangeLogLocation $packageProperties.ChangeLogPath -VersionString $newVersion -ForRelease $true
+
+if (!$changelogIsValid)
+{
+  Write-Host "The changelog [$($packageProperties.ChangeLogPath)] is not valid for release. Please make sure it is valid before queuing release build." -ForegroundColor Red
```

Comment:
Should we just call Write-Warning here instead of setting the test color to Red?

Reply:
Propably better.